pkg/apis/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rook/rook/pkg/apis
 
-go 1.22.6
+go 1.22.5
```

Comment:
should we run `go mod tidy` ?

Reply:
I have run already otherwise build CI would have failed

Reply:
Okay, I see rook is utilizing the CSI addons API,
And it is missing in the rook API folder, which is correct.....